documentation/keys.md

```diff
@@ -0,0 +1,79 @@
+# Keys
+A detailed description of each key is listed below. These keys will be listed in the order they appear on the
+settings page. The "Alternate Functionality" listed below a feature can be activated by holding **SHIFT +**
+the key of the feature.
+## Move
+When looking at a region page, pressing this key will move your nation to the region.
+
+When looking at the reports page,
+pressing this key will refresh the page and click the region the raiders moved to, if applicable.
```

Comment:
Minor nitpick, but I feel like the wording could be read as suggesting the keybind triggers both actions when it should only trigger one.
```suggestion
pressing this key will click the region the raiders moved to, if applicable, or refresh the page otherwise.
```